apps/dashboard/pkg/migration/schemaversion/migrations.go

```diff
@@ -48,11 +41,11 @@ func GetMigrations(dsInfoProvider DataSourceInfoProvider, panelProvider PanelPlu
 		21: V21,
 		22: V22,
 		23: V23,
-		24: V24(panelProvider),
+		24: V24,
 		25: V25,
 		26: V26,
 		27: V27,
-		28: V28(panelProvider),
```

Comment:
Technically we could even refactor v28 migrator to not have be a factory, and follow all the other migrators pattern except the ones with datasources